Cargo.toml

```diff
@@ -30,6 +30,8 @@ tar = "0.4"
 flate2 = "1.0"
 daemonize = "0.4.1"
 snap = "1.0.5"
+assert_cmd = "2.0.4"
```

Comment:
Can you move these to the `dev.dependencies` because they are not needed to build the binary. But only during development (and testing)